code/game/objects/structures.dm

```diff
@@ -77,18 +78,18 @@
 	src.add_fingerprint(user)
 	user.visible_message("<span class='warning'>[user] starts climbing onto [src].</span>", \
 								"<span class='notice'>You start climbing onto [src]...</span>")
-	var/climb_time = 20
+	var/adjusted_climb_time = climb_time
 	if(user.restrained()) //climbing takes twice as long when restrained.
-		climb_time *= 2
+		adjusted_climb_time *= 2
 	structureclimber = user
-	if(do_mob(user, user, climb_time))
+	if(do_mob(user, user, adjusted_climb_time))

```

Comment:
Isn't it a little redundant to use another var when it's identical to the original?